CovertInfiltration/Src/CovertInfiltration/Classes/UIAvengerHUD_CAButton.uc

```diff
@@ -12,25 +12,28 @@ var localized string CovertActionButtonLabel;
 
 event OnInit(UIScreen Screen)
 {
-
 	local UILargeButton CovertActionButton;
+	local UILargeButton CovertActionButtonNew;
 	local UIStrategyMap StrategyMap;
 
 	StrategyMap = UIStrategyMap(Screen);
 	if (StrategyMap == none) return;
 
 	CovertActionButton = Screen.Spawn(class 'UILargeButton', StrategyMap.StrategyMapHUD);
-
 	CovertActionButton.InitButton('CovertActionButton', CovertActionButtonLabel, OnCovertActionButton);
 	CovertActionButton.AnchorBottomCenter();
 	CovertActionButton.SetFontSize(40);
 	CovertActionButton.SetPosition(-135, -200);
-	
+
+	CovertActionButtonNew = Screen.Spawn(class 'UILargeButton', StrategyMap.StrategyMapHUD);

```

Comment:
The CovertActionButtonNew should take the place of the original CovertActionButton now that its interface is fully functional.